interface_frontend.py

```diff
@@ -188,6 +190,27 @@ def draw_interface(interface_state, window):
                 ):
             draw_card(coordinate, card)
 
+    if interface_state.players:
+        # Other robots background
+        for i in range(len(interface_state.players)):
+            players_background.x = 50 + i * 98
+            players_background.y = 50
+            players_background.draw()
+
+        # Other robots
+        image_coordinates = []
```

Comment:
Tohle zase není potřeba dávat do seznamu – můžeš to spočíst až když to budeš potřebovat.

Reply:
jenže jsem nedošla na to kde jak :( ano vidím že to nastavuju níž, ale nevím kde teda to "i" napsat v nějakém cyklu aby mi to spočítalo správně, protože se mi potom ti roboti vykreslovali vícekrát. Můžeš mi víc poradit? @encukou

Reply:
Zkus si:
```
>>> robots = ['🔵', '🤖', '🔷']
>>> list(enumerate(robots))
[(0, '🔵'), (1, '🤖'), (2, '🔷')]
>>> for i, robot in enumerate(robots):
...     print(i, robot)
... 
0 🔵
1 🤖
2 🔷
```

Reply:
prostě že nevím kde jak navyšovat to "i" . takto to sice funguje, ale zase jsi říkal, že nemám to i dávat do zipu že?
```
for (robot, i) in zip (interface_state.players, range(len(interface_state.players))):
            if robot.name in loaded_robots_images.keys():
                player_sprite.image = loaded_robots_images[robot.name]
                player_sprite.x = 60 + i * 98
                player_sprite.y = 85
                player_sprite.draw()
```

Reply:
aha díky, já s tím enumerate právě neumím pracovat, díky vyzkouším si.